package.json

```diff
@@ -4,7 +4,8 @@
   "version": "0.0.1",
   "scripts": {
     "start": "HOT_RELOAD=1 webpack-dev-server --mode development",
-    "lint": "eslint .",
+    "start:docker": "HOT_RELOAD=1 webpack-dev-server --mode development --host 0.0.0.0",
```

Comment:
does it hurt to just combine this with `start`.  I can't think of a reason that running `--host 0.0.0.0` for `npm start` hurts anything.